.travis.yml

```diff
@@ -96,4 +103,4 @@ branches:
     - master
 
 notifications:
-  email: false
+  email: false
```

Comment:
could you add back the end-of-file new-line?

Reply:
Done